jax/experimental/jax2tf/tests/correctness_stats.py

```diff
@@ -69,16 +70,59 @@ def _to_np_dtype(dtype):
       tf_unimpl(f"{prim.name} is unimplemented for dtype {np_dtype}")
 
   if prim in [lax.rem_p, lax.atan2_p]:
-    # b/158006398: TF kernels are missing for 'rem' and 'atan2'
     np_dtype = _to_np_dtype(args[0].dtype)
     if np_dtype in [np.float16, dtypes.bfloat16]:
+      # b/158006398: TF kernels are missing for 'rem' and 'atan2'
       tf_unimpl(f"Missing TF kernels for {prim.name} with dtype {np_dtype}")
 
   if prim is lax.nextafter_p:
     np_dtype = _to_np_dtype(args[0].dtype)
     if np_dtype in [np.float16, dtypes.bfloat16]:
       tf_unimpl(f"{prim.name} is unimplemented for dtype {np_dtype}")
 
+  if prim is lax_linalg.qr_p:
+    np_dtype = _to_np_dtype(args[0].dtype)
+    if np_dtype in [np.complex64, np.complex128]:
+      # See https://github.com/google/jax/pull/3775#issuecomment-659407824;
+      # experimental_compile=True breaks for complex types.
+      tf_unimpl(f"{prim.name} is unimplemented for dtype {np_dtype}")
+
+  if prim is lax_linalg.svd_p:
+    np_dtype = _to_np_dtype(args[0].dtype)
+    if np_dtype in [np.float16, dtypes.bfloat16]:
+      # TODO: SVD on TPU for bfloat16 seems to work for JAX but fails for TF
+      tf_unimpl(f"{prim.name} is unimplemented for dtype {np_dtype}",
+                devs=["TPU"])
+    elif np_dtype in [np.complex64, np.complex128]:
+      # TODO: on CPU and GPU "No registered 'Svd' OpKernel for XLA_CPU_JIT
```

Comment:
I am wondering if the `Works on JAX because JAX uses a custom implementation` deserves to be in the generated documentation?

Reply:
Added it in `c289a06`